pkg/controller/atlasdeployment/backup.go

```diff
@@ -276,26 +290,87 @@ func (r *AtlasDeploymentReconciler) updateBackupScheduleAndPolicy(
 	return nil
 }
 
-func backupSchedulesAreEqual(currentSchedule *mongodbatlas.CloudProviderSnapshotBackupPolicy, newSchedule *mongodbatlas.CloudProviderSnapshotBackupPolicy) (bool, error) {
+func backupSchedulesAreEqual(currentSchedule *mongodbatlas.CloudProviderSnapshotBackupPolicy, newSchedule *mongodbatlas.CloudProviderSnapshotBackupPolicy) (BackusScheduleCompareResult, error) {
 	currentCopy := mongodbatlas.CloudProviderSnapshotBackupPolicy{}
 	err := compat.JSONCopy(&currentCopy, currentSchedule)
 	if err != nil {
-		return false, err
+		return bsNotEqual, err
 	}
 
 	newCopy := mongodbatlas.CloudProviderSnapshotBackupPolicy{}
 	err = compat.JSONCopy(&newCopy, newSchedule)
 	if err != nil {
-		return false, err
+		return bsNotEqual, err
 	}
 
 	normalizeBackupSchedule(&currentCopy)
 	normalizeBackupSchedule(&newCopy)
+
+	// Should never happend because the must be at least one policy in Atlas
+	if len(currentCopy.Policies) == 0 {
+		return bsEqual, nil
+	}
+
+	defaultBs := mongodbatlas.CloudProviderSnapshotBackupPolicy{
+		ClusterID:             currentCopy.ClusterID,
+		ClusterName:           currentCopy.ClusterName,
+		ReferenceHourOfDay:    toptr.MakePtr[int64](12),
+		ReferenceMinuteOfHour: toptr.MakePtr[int64](19),
+		RestoreWindowDays:     toptr.MakePtr[int64](2),
+		Policies: []mongodbatlas.Policy{
+			{
+				ID: currentCopy.Policies[0].ID,
+				PolicyItems: []mongodbatlas.PolicyItem{
+					{
+						FrequencyInterval: 6,
+						FrequencyType:     "hourly",
+						RetentionUnit:     "days",
+						RetentionValue:    2,
+					},
+					{
+						FrequencyInterval: 1,
+						FrequencyType:     "daily",
+						RetentionUnit:     "days",
+						RetentionValue:    7,
+					},
+					{
+						FrequencyInterval: 6,
+						FrequencyType:     "weekly",
+						RetentionUnit:     "weeks",
+						RetentionValue:    4,
+					},
+					{
+						FrequencyInterval: 40,
+						FrequencyType:     "monthly",
+						RetentionUnit:     "months",
+						RetentionValue:    12,
+					},
+				},
+			},
+		},
+		AutoExportEnabled:                 toptr.MakePtr(false),
+		UseOrgAndGroupNamesInExportPrefix: toptr.MakePtr(false),
+		CopySettings:                      []mongodbatlas.CopySetting{},
+	}
```

Comment:
I would make this a variable or external function, it now makes the current function code longer and more difficult to read.